src/AgreementEligibility.sol

```diff
@@ -236,6 +241,24 @@ contract AgreementEligibility is HatsEligibilityModule {
     /// @dev Hats.sol will emit a Hats.WearerStandingChanged event
   }
 
+  /**
+   * @notice Set a new owner hat
+   * @dev Only callable by a wearer of the current ownerHat, and only if the target hat is mutable
+   * @param _newOwnerHat The new owner hat
+   */
+  function setOwnerHat(uint256 _newOwnerHat) public onlyOwner hatIsMutable {
+    _setOwnerHat(_newOwnerHat);
+  }
+
+  /**
+   * @notice Set a new arbitrator hat
+   * @dev Only callable by a wearer of the current arbitratorHat, and only if the target hat is mutable
```

Comment:
Documentation fix: "ownerHat" instead of "arbitratorHat". Or is the intention here is to provide the permission to the arbitrator?